app/containers/App/tests/index.test.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
+import ProgressBar from 'components/ProgressBar';
 import Header from 'components/Header';
 import Footer from 'components/Footer';
 import App from '../index';
```

Comment:
Well, isn't his fun, the Github online editor returns 404 on this file...

The problem created by merging `dev` is:

`containers/App/tests/index.test.js`
```javascript
import App from '../index';
```
Needs to change to:
```javascript
import { App } from '../index';
```

I don't understand why the braces are necessary considering this from `../index`:
```javascript
export default withRouter(App);
```

Something strange about the `withRouter` wrapper? But that's how it was when it worked and it works (for me) when I add the braces.